components/forms/TextInput/TextInput.tsx

```diff
@@ -6,7 +6,7 @@ import { InputFieldProps, CharacterCountMessages, HTMLTextInputTypeAttribute } f
 
 export interface TextInputProps extends InputFieldProps {
   type: HTMLTextInputTypeAttribute;
-  characterCountMessages: CharacterCountMessages;
+  characterCountMessages?: CharacterCountMessages;
```

Comment:
I aimed for the most minimal change -vs- a big refactor or a new component

Reply:
This works well for now thanks @thiessenp-cds